scipy/stats/tests/test_stats.py

```diff
@@ -7087,6 +7087,27 @@ def test_weights_masked_1d_array(self, xp):
         check_equal_gmean(a, desired, weights=weights, rtol=1e-5,
                           dtype=np.float64, xp=xp)
 
+    #@skip_xp_backends('jax.numpy', reason="JAX doesn't allow item assignment.")
+    @pytest.mark.parametrize('axis', [0, 1])
+    def test_marray(self, axis):
+        xp = np  # can't use array-api-compat xp because data-apis/array-api-compat#226
+        try:
+            import marray
+            mxp = marray._get_namespace(xp)
+        except ModuleNotFoundError:
```

Comment:
I think this can be tidied up a little bit with `pytest.importorskip`

Reply:
Thanks. Didn't know about that!

Reply:
So with that, you think this pattern is OK? Add new test to exercise masked arrays, and run for all `xp` backends?

Reply:
No objections, we can always reconsider if it becomes too much for a burden